components/ai_chat/resources/page/components/conversation_list/style.module.scss

```diff
@@ -4,45 +4,102 @@
 // you can obtain one at https://mozilla.org/MPL/2.0/.
 
 .list {
-  padding: 16px;
-}
-
-.textBase {
-  p {
-   margin: 0;
-   white-space: pre-wrap;
+  div[data-turn='assistant'] {
+    padding-bottom: 0;
   }
- }
+}
 
 .turnAI {
-  @extend .textBase;
+  display: flex;
+  align-items: flex-start;
+  gap: 16px;
+  font: var(--leo-font-secondary-default-regular);
+  color: var(--leo-color-text-primary);
   padding: 16px;
 }
 
 .turnHuman {
-  @extend .textBase;
-  padding: 0 16px 16px 0;
-
-  // bubble wrap
-  p:first-child {
-    background-color: var(--leo-color-container-highlight);
-    padding: 16px;
-    border-radius: 16px 16px 2px 16px;
-  }
+  display: flex;
+  align-items: flex-start;
+  gap: 16px;
+  font: var(--leo-font-secondary-default-regular);
+  color: var(--leo-color-text-primary);
```

Comment:
nit: Instead of repeating these you could add a selector for `.turnHuman, .turnAI`, or add a class name for `.turn`

Reply:
done